src/runtime_src/core/pcie/emulation/hw_emu/alveo_shim/shim.cxx

```diff
@@ -1562,7 +1562,7 @@ namespace xclhwemhal2 {
     //Need to be removed when NOC allows transaport_dbg access
     if (xclemulation::config::getInstance()->isFastNocDDRAccessEnabled())
     {
-      std::cout << "Checking Read Fastmem " << src << std::endl;
+      // std::cout << "Checking Read Fastmem " << src << std::endl;
       if (mNocFastAccess.isAddressMapped(src, size))
       {
         mNocFastAccess.read(src, (unsigned char *)dest, size);
```

Comment:
Is there any reason why we did not remove this std::cout? Please remove even this

Reply:
Removed.

Reply:
I see its commented. I think its better to remove

Reply:
Done, removed all the 4 lines having unnecessary cout.